code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1771,17 +1799,57 @@
 	reagent_state = LIQUID
 	color = "#C8A5DC" // rgb: 200, 165, 220
 
-/datum/reagent/dexalinp/on_mob_life(var/mob/living/M as mob)
+/datum/reagent/dexalinp/on_mob_life(var/mob/living/M as mob, var/alien)
+	if(!holder) return
+	if(!M) M = holder.my_atom
+
+	if(M.stat == DEAD)
+		return
+	if(alien && alien == IS_VOX)
+		M.adjustToxLoss(REAGENTS_METABOLISM * 2)
+	else
+		M.adjustOxyLoss(-M.getOxyLoss())
+		if(holder.has_reagent("lexorin"))
+			holder.remove_reagent("lexorin", 2 * REM)
+		..()
 
+/datum/reagent/vexalin
+	name = "Vexalin"
+	id = "vexalin"
+	description = "Vexalin is used in the treatment of nitrogen deprivation."
+	reagent_state = LIQUID
+	color = "#FAA5C8" // rgb: 250, 165, 200
+
+/datum/reagent/vexalin/on_mob_life(var/mob/living/M as mob, var/alien)
 	if(!holder) return
-	if(M.stat == 2.0)
+	if(!M) M = holder.my_atom
+
+	if(!alien || alien != IS_VOX)	//Vox masterrace, SKREEEEEEEEEEEE!
 		return
+
+	M.adjustOxyLoss(-2 * REM)
+	if(holder.has_reagent("lexorin"))
+		holder.remove_reagent("lexorin", 2 * REM)
+	..()
+
+/datum/reagent/vexalinp
+	name = "Vexalin Plus"
+	id = "vexalinp"
+	description = "Dexalin Plus is used in the treatment of nitrogen deprivation. Its highly effective."
```

Comment:
Typo? 
	description = "Vexalin Plus is used in the treatment of nitrogen deprivation. Its highly effective."